### ISSUE TITLE
Enable Airflow login page

### ISSUE THREAD

**Issue by andresionek91** [andresionek91/airflow-autoscaling-ecs#11]
Enable Airflow login page and save user credentials to secret manager

**Comment by mkwatson** [comment]
Yeah, this seems sort of useless with an airflow webserver that's open to the world. We really want to use this!

**Comment by mkwatson** [comment]
For now I'm using google auth
https://airflow.apache.org/docs/stable/security.html#google-authentication

**Comment by andresionek91** [comment]
Yep, I'm planning to enable different auth methods soon. 

So far you can restrict the access by whitelisting IPs here: https://github.com/andresionek91/airflow-autoscaling-ecs/blob/master/service.yml#L84-L85

Better than open to the world :)
